docs/user/user_registration.md

```diff
@@ -1,16 +1,54 @@
 # User Registration
 
 Before you can begin to manage your projects and access your data within the data-manager, 
-you need to create an account for the platform. For this you need to perform the following steps.
+you need to create an account for the platform. We currently support 2 modes of registration:
 
-1. [Register an account within the data manager](#register-account)
-2. [Validate the created account via the validation link you received in your email](#validate-the-registered-account)
-3. [Login into the data manager platform](#login-to-the-data-manager)
+1. [Register an account via orcid](#register-an-account-via-orcid)
+2. [Register an account via email](#register-an-account-via-email)
 
-## Register Account
+## Register an account via orcid
 
-From the login page you can navigate to the registration view via the "registration" button on the top right 
-or the registration link on the bottom of the login
+### Login to orcid
+
+We recommend to register to the data manager via your existing [orcid](https://orcid.org) account.
```

Comment:
That's assuming a lot. Since you explain the process of creating a new orcid below, maybe just say "via an orcid account"

Reply:
also maybe explain what the benefit is. the process looks more complicated, but in the end the user does not have to type in any credentials any more as long as they are still logged into the orcid page.

Reply:
Good point thanks!